crypto3/libs/blueprint/include/nil/blueprint/zkevm_bbf/zkevm.hpp

```diff
@@ -510,7 +516,7 @@ namespace nil {
                                     if( constraints.size() <= i) continue;
                                     acc_constraint += context_object.relativize(zkevm_opcode_row_selectors[pair], -1) * constraints[i];
                                     //std::cout << "\topcode " << pair.first << " row " << pair.second << " constraint " << context_object.relativize(zkevm_opcode_row_selectors[pair], -1) * constraints[i] << std::endl;
-                                    //relative_mc.push_back(context_object.relativize(zkevm_opcode_row_selectors[pair], -1) * constraints[i]);
+                                    relative_mc.push_back(context_object.relativize(zkevm_opcode_row_selectors[pair], -1) * constraints[i]);
```

Comment:
Why are we doing it?

Reply:
We should comment it again. It was uncommented for testing.
These constraints may be applied and tested separately and sometimes it's more convenient for debugging.

But we already append them to acc_constraint.
It's enough.